.github/workflows/new-release.yml

```
@@ -0,0 +1,48 @@
+        with:
+          toolchain: stable
+      - run: cargo build --locked --release --features memmap2 --target ${{ matrix.platform.target }}
+      - run: |
+          cd target/${{ matrix.platform.target }}/release
+          ARCHIVE=rcr_${{ matrix.platform.target }}.zip
+          zip -9r ${ARCHIVE} ${{ matrix.platform.binary }}
+          printf "%s %s" "$(sha256sum "${ARCHIVE}" | cut -d ' ' -f 1)" "$FILE_NAME" > "${ARCHIVE}.sha256sum"
+          cd -
+      # do this for the moment for testing
+      - name: Publish release artifacts
+        uses: actions/upload-artifact@v3
```

actionlint (1.7.10)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)